logback/src/main/java/com/linecorp/armeria/common/logback/BuiltInProperty.java

```diff
@@ -157,8 +161,16 @@
         mdcKeyToEnum = builder.build();
     }
 
-    static Optional<BuiltInProperty> findByMdcKey(String mdcKey) {
-        return Optional.ofNullable(mdcKeyToEnum.get(mdcKey));
+    static List<BuiltInProperty> findByMdcKeyPattern(String mdcKeyPattern) {
+        final Pattern pattern = Pattern.compile(
+                ("\\Q" + mdcKeyPattern + "\\E").replace(WILDCARD_STR, "\\E.*\\Q"));
```

Comment:
Perhaps we should use `replaceAll()` to handle the case where there are more than one asterisk in the pattern?

Reply:
`String.replace` replaces matched targets, so it is working well as expected. But it seems to be good to avoid confusion. I will change it to `replaceAll`.